src/libraries/System.Private.Xml/src/System/Xml/Xsl/IlGen/XmlILModule.cs

```diff
@@ -239,7 +239,7 @@ public void BakeMethods()
 
             if (!_useLRE)
             {
-                typBaked = _typeBldr!.CreateTypeInfo()!.AsType();
+                typBaked = _typeBldr.CreateType();
```

Comment:
```suggestion
                typBaked = _typeBldr!.CreateType();
```
I think this needs to stay.

Reply:
Added back the ! to `_typeBldr`.